Tests/StreamChatUITests/AssertSnapshot.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import SnapshotTesting
+import XCTest
+
+/// Snapshot of a view controller. All variants will be tested by default. For each variant, it will take a snapshot.
+/// It is recommended to test all trait variants only for view controllers, and for small view components only test
+/// the default variant. This is to minimize the amount of pictures stored in the repo.
+///
+/// - Parameters:
+///   - vc: The view controller to be tested.
+///   - isEmbededInNavigationController: If the view controller should be embeded in a navigation controller.
+///   Useful to test the navigation bar.
+///   - variants: The variants that a snapshot will be taken. All variants should be tested for a view controller.
+///   - record: True if a new reference should be saved. False by default,
+///   so that the newly captured snapshot is compared with the current reference.
+func AssertSnapshot(
+    _ vc: UIViewController,
+    isEmbededInNavigationController: Bool = false,
+    variants: [SnapshotVariation] = SnapshotVariation.allCases,
+    record: Bool = false,
+    line: UInt = #line,
+    file: StaticString = #file,
+    function: String = #function
+) {
+    let viewController = isEmbededInNavigationController ? UINavigationController(rootViewController: vc) : vc
+    variants.forEach { variant in
+        assertSnapshot(
+            matching: viewController,
+            as: .image(size: CGSize(width: 360, height: 700), traits: variant.traits),
```

Comment:
this size should be in a constant somewhere. See similar constants like `defaultTimeout`...